employeeapplication/src/main/java/com/interview/employeeapplication/service/EmployeeService.java

```
@@ -2,11 +2,12 @@
 import com.interview.employeeapplication.entity.Employee;
 
 import java.util.List;
+import java.util.Optional;
 
 
-  public interface EmployeeService {
+public interface EmployeeService {
         List<Employee> getAllEmployees();
-        Employee getEmployeeById(Long id);
+      Optional<Employee> getEmployeeById(Long id);
```

throw exception if employee not found instead of returning optional

---

I have fixed as you have suggested.
Please refer this PR#10
https://github.com/bluesnowArray/employee-application-project/pull/10/commits/38a0e8aeabd7b165eec4a11ff6982866e6984741